src/test/java/ebl/v2/ShippingInstructionIT.java

```diff
@@ -61,14 +64,9 @@ void testFailureOnEmptyBodyForPostShippingInstruction() throws JsonProcessingExc
         .then()
         .assertThat()
         .statusCode(HttpStatus.SC_BAD_REQUEST)
-        .body("httpMethod", equalTo("POST"))
-        .body("requestUri", containsString("/v2/shipping-instructions"))
-        .body("errors[0].reason", equalTo("invalidInput"))
-        .body("errors[0].message", containsString("isToOrder"))
-        .body("errors[0].message", containsString("shipmentEquipments"))
-        .body("errors[0].message", containsString("isShippedOnboardType"))
-        .body("statusCode", equalTo(400))
-        .body("statusCodeText", equalTo("Bad Request"))
+        .body("status", equalTo(400))
```

Comment:
Have you run this locally? The reason httpMethod etc were missing was because of a missing annotation for documentStatus, Created/UpdatedDateTime, but that has since been fixed. This change should fail.

Reply:
I can see that now, missed it since my event-core dependency was not updated. 

Will fix (y)